service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/PopulateStandardDirectionsHandler.java

```diff
@@ -117,20 +99,25 @@ public void populateStandardDirections(PopulateStandardDirectionsEvent event) th
         return caseDetails.getData();
     }
 
-    private LocalDateTime getCompleteByDate(CaseData caseData, DirectionConfiguration direction) {
-        LocalDateTime completeBy = null;
+    private LocalDateTime getFirstHearingStartDate(List<Element<HearingBooking>> hearings) {
+        return hearingBookingService.getFirstHearing(hearings)
+            .map(HearingBooking::getStartDate)
+            .orElseThrow(() -> new IllegalStateException("Expected to have at least one hearing booking"));
```

Comment:
Maybe it would make sense to add new exception for that, we throw the same exception in `HearingBookingService`

Reply:
chatted this through with Adam, this is fine for now.